.pylintrc

```diff
@@ -0,0 +1,577 @@
+[THINGS WE CARE ABOUT]
+
+[FORMAT]
+
+# Expected format of line ending, e.g. empty (any line ending), LF or CRLF.
+expected-line-ending-format=
+# Naming style matching correct class names.
+class-naming-style=PascalCase
+
+# Maximum number of nested blocks for function / method body
+max-nested-blocks=5
+
+# Maximum number of characters on a single line.
+max-line-length=90

```

Comment:
What do we want for max-line-length?

Reply:
ESLint defaults to 100, I believe the default in Vim is 80, somewhere in that ballpark. 90 is fine, imo.

Reply:
@rconn478 thots?

Reply:
90 seems like a good compromise.